src/averageAges.js

```diff
@@ -55,7 +93,26 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  let count = 0;
```

Comment:
Use a more descriptive variable name instead of `count`. Consider using `ageDifferenceCount` for better readability.

Reply:
You might event not need it and use array.length property to access total count of items in resulting array